plugins/GSdx/GLLoader.cpp

```diff
@@ -367,7 +367,12 @@ namespace GLLoader {
 #ifdef _WIN32
 		if (status) {
 			if (fglrx_buggy_driver) {
-				fprintf(stderr, "OpenGL renderer is slow on AMD GPU due to inefficient driver. Sorry.\n");
+				fprintf(stderr, "The \"OpenGL Hardware\" renderer is slow on AMD GPUs due to an inefficient driver. Check out the links below for further information.\n"
```

Comment:
``Hardware`` **->** ``hardware``. Also I think the quotes surrounding ``OpenGL Hardware`` is unnecessary here, better to remove them?